common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -497,4 +497,15 @@ trait FeatureSwitches {
     exposeClientSide = true
 
   )
+
+  val ArticleWithStructuredRecipe = Switch(
+    SwitchGroup.Feature,
+    "is-article-with-structured-recipe-data",
+    "changes design of articles with strucutred recipe data",
+    owners = Seq(Owner.withGithub("tsop14"), Owner.withGithub("blongden73")),
+    safeState = Off,
+    sellByDate = new LocalDate(2017, 12, 1),
```

Comment:
In case of a feature switch, the convention is to pick a reasonable date where we expect the feature to be fully ready (or removed if tests are unsuccessful).
The idea behind a feature switch is to give us more control in order to enable/disable a feature while it is behind developed and tested. The `sellByDate` is here to ensure we don't sit on a half-baked feature for too long and force us to reevaluate it.

In your case I would set the date to 2-3 weeks. That would give you plenty of time to test and finalize this feature while keeping the lifetime of the switch as short as possible. 

Does it make sense?